### DISCUSSION TITLE
Disable TOC via frontmatter?

### DISCUSSION THREAD

**Discussion by nousernameavailableanymore** [mkdocs/mkdocs#2486] [Q&A]
Is it possible to disable the TOC on the left side on a per-page basis via yml frontmatter or something like this?
I found a CSS-"hack" to disable the TOC for all pages, but I m wondering if this can be set for each page independently.
Thank you.

**Comment by oprypin** [comment]
There is an ability specific to [mkdocs-material theme](https://squidfunk.github.io/mkdocs-material/):
https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#hiding-the-sidebars

But sounds like what you're describing *is* about the mkdocs-material theme, so hopefully that's your answer.

**Reply by oprypin** [reply]
For reference, this is where it's implemented: https://github.com/squidfunk/mkdocs-material/blob/68de4db94d0f71ec4aa6d364435eebd7c3fa74f7/material/base.html#L131

**Reply by nousernameavailableanymore** [reply]
I m using the bootswatch Superhero theme.
So I assume its not possible on this theme then.
